tools/postman/scripts/compare-forks.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -o errexit
```

Comment:
```suggestion
set -o errexit
```

Reply:
I will change this for all scripts in a different PR. Thank you for all the comments!